spring-cloud-dataflow-docs/src/main/asciidoc/dashboard.adoc

```diff
@@ -3,330 +3,379 @@
-composed tasks.
+
+The dashboard includes the Create Composed Task tab, which provides an interactive graphical interface for creating composed tasks.
 
 In this tab, you can:
 
-* Create and visualize composed tasks using DSL, a graphical canvas, or both
-* Use auto-adjustment and grid-layout capabilities in the GUI for simpler and
-interactive organization of the composed task
+* Create and visualize composed tasks using DSL, a graphical canvas, or both.
+* Use auto-adjustment and grid-layout capabilities in the GUI for simpler and interactive organization of the composed task.
+
+On the Create Composed Task screen, you can define one or more task parameters by entering both the parameter key and the parameter value.
+
+NOTE: Task parameters are not typed.
+
+The following image shows the composed task designer:
 
 .Composed Task Designer
 image::{dataflow-asciidoc}/images/dataflow-ctr-flo-tab.png[Composed Task Designer, scaledwidth="100%"]
 
-==== Launching Tasks
 
-Once the task definition is created, they can be launched through the Dashboard
-as well. Navigate to the *Definitions* tab. Select the Task you want to launch by
-pressing `Launch`.
 
-On the following screen, you can define one or more Task parameters by entering:
+==== Launching Tasks
+
+Once the task definition has been created, the tasks can be launched through the dashboard.
+To do so, click the Definitions tab and select the task you want to launch by pressing `Launch`.
 
-* Parameter Key
-* Parameter Value
 
-Task parameters are not typed.
 
 [[dashboard-tasks-executions]]
 === Executions
 
+The Executions tab shows the current running and completed tasks.
+
+The following image shows the Executions tab:
+
 .List of Task Executions
 image::{dataflow-asciidoc}/images/dataflow-task-executions-list.png[List of Task Executions, scaledwidth="100%"]
 
+
+
 [[dashboard-jobs]]
 == Jobs
 
-The _Jobs_ section of the Dashboard allows you to inspect *Batch Jobs*. The main
-section of the screen provides a list of Job Executions. *Batch Jobs*
-are *Tasks* that were executing one or more *Batch Job*. As such each
-Job Execution has a back reference to the *Task Execution Id* (Task Id).
+The Jobs section of the Dashboard lets you inspect batch jobs.
+The main section of the screen provides a list of job executions.
+Batch jobs are tasks that each execute one or more batch jobs.
+Each job execution has a reference to the task execution ID (in the Task Id column).
+
+The list of Job Executions also shows the state of the underlying Job Definition.
+Thus, if the underlying definition has been deleted, "`deleted`" appears in the Status column.
 
-In case of a failed job, you can also restart the task. When dealing with long-running
-Batch Jobs, you can also request to stop it.
+You can take the following actions for each job:
+
+* Restart (for failed jobs).
+* Stop (for running jobs).
+* View execution details.
+
+// TODO To get to the Job Execution Details page, what does one click?
```

Comment:
You need to deploy a batch-job in SCDF to get that link enabled. The jobs-tab will also show that record.